web/serverconfig.sample.php

```diff
@@ -10,6 +10,8 @@
 define('YAAMP_DBNAME', 'yaamp');
 define('YAAMP_DBUSER', 'root');
 define('YAAMP_DBPASSWORD', 'password');
+define('YAAMP_LOGS', '/var/log/yiimp');
```

Comment:
YAAMP_LOGS is already defined in the same file at line 5:
https://github.com/tpruvot/yiimp/blob/next/web/serverconfig.sample.php#L5